src/shared/components/comment/comment-node.tsx

```diff
@@ -282,7 +282,7 @@ export class CommentNode extends Component<CommentNodeProps, CommentNodeState> {
 
     return (
       <li className="comment" role="comment">
-        <div
+        <article
```

Comment:
Are article tags usually used for comments in other sites using semantic tags?

Reply:
> Are article tags usually used for comments in other sites using semantic tags?

`blockquote` is what's used by kbin although I don't think that's better because then quotes are also `blockquote`.

`section` might be better

Reply:
I was following the advice I see here, thoughts?

https://www.w3schools.com/html/html5_semantic_elements.asp

![image](https://github.com/LemmyNet/lemmy-ui/assets/3112763/4906025c-9d4d-42dc-8da3-f86e12feaf1d)

Reply:
@Zetaphor looks good. I was asking more because I wasn't sure the correct tag to use rather than saying you were using it wrong.

Reply:
I wasn't sure either, this is my first go at properly using semantic tags. If it's good with everyone I'll mark this one resolved.